Themes/default/css/index.css

```diff
@@ -1174,17 +1185,31 @@ img#smflogo {
 	clear: both;
 }
 /* The upper_section, float the two each way */
+#inner_wrap {
+	display: flex;
+	justify-content: space-between;
+	border-bottom: 1px solid #bbb;
```

Comment:
This does not look good for guests on mobile (<480px)

Reply:
Fixed in acfa658